workflows/README.md

```diff
@@ -4,6 +4,10 @@ The content in this folder doesn't _necessarily_ belong in this repository, but
 
 Example implementation of these workflow files [can be found in `exercism/javascript`][git-javascript].
 
+## **HELP** this looks like a lot of work
+
+The rest of the document is really to explain what's going on in the workflows. If you're in a hurry, and you really want to drop `.travis.yml`, but you can't invest right now in making the PR scripts are optimized, scroll down to **Migrating from Travis**.
```

Comment:
```suggestion
The rest of the document is really to explain what's going on in the workflows. If you're in a hurry, and you really want to drop `.travis.yml`, but you can't invest right now in making the PR scripts are optimized, scroll down to [**Migrating from Travis**](#migrating-from-travis).
```

(not 100% sure this is the right syntax)

Reply:
(not sure either, but we can try)